audit-test-tools/template.yaml

```diff
@@ -0,0 +1,126 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Audit accoun test tool infrastructure
+Transform: AWS::Serverless-2016-10-31
+
+Parameters:
+  CodeSigningConfigArn:
+    Description: The ARN of the Code Signing Config to use, provided by the deployment pipeline
+    Type: String
+    Default: none
+  Environment:
+    Description: The environment type
+    Type: String
+    AllowedValues:
+      - dev
+      - build
+      - staging
+  PermissionsBoundary:
+    Description: The ARN of the permissions boundary to apply to any role created by the template
+    Type: String
+    Default: none
+
+Conditions:
+  UseCodeSigning: !Not [!Equals [!Ref CodeSigningConfigArn, none]]
+  UsePermissionsBoundary: !Not [!Equals [!Ref PermissionsBoundary, none]]
+
+Globals:
+  Function:
+    CodeSigningConfigArn:
+      !If [UseCodeSigning, !Ref CodeSigningConfigArn, !Ref AWS::NoValue]
+    CodeUri: dist/
+    Environment:
+      Variables:
+        NODE_OPTIONS: --enable-source-maps
+    PermissionsBoundary:
+      !If [UsePermissionsBoundary, !Ref PermissionsBoundary, !Ref AWS::NoValue]
+    ReservedConcurrentExecutions: 10
+    Runtime: nodejs16.x
+    Timeout: 30
+
+Resources:
+  WriteTestDataToAthenaBucketFunction:
```

Comment:
Do we need an SSM param to output the name so other stacks can use?

Reply:
No, because we're interacting via a queue